doc/build-osx.md

```diff
@@ -51,7 +51,7 @@ Running
 
 Dash Core is now available at `./src/dashd`
 
-Before running, it's recommended that you create an RPC configuration file.
+Before running, you may create an empty configuration file.
 
     echo -e "rpcuser=dashrpc\nrpcpassword=$(xxd -l 16 -p /dev/urandom)" > "/Users/${USER}/Library/Application Support/DashCore/dash.conf"
```

Comment:
14684 this needs changing

https://github.com/bitcoin/bitcoin/pull/14684/files#diff-e9ba5bd8b63e72aa2d54d02960c9209823938981691cc8786ba323b54de0659cL85-R85